.github/ISSUE_TEMPLATE/bug.yml

```
@@ -0,0 +1,63 @@
+name: Bug report
+description: Submit a bug report
+labels: ["type-bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+          # New to Python?
+
+          The issue tracker isn't the right place to get help.
+          Consider instead:
+
+            - reading the [Python tutorial](https://docs.python.org/3/tutorial/)
+            - posting on [Discourse](https://discuss.python.org/c/users/7)
+            - emailing [python-list](https://mail.python.org/mailman/listinfo/python-list)
+  - type: checkboxes
+    attributes:
+      label: Checklist
+      description: Bugs in third-party projects (e.g. `requests`) do not belong in the CPython issue tracker
+      options:
+        - label: I am confident this is a bug in CPython, not a bug in a third-party project
+          required: true
+        - label: |
+            I have searched the [CPython issue tracker](https://github.com/python/cpython/issues),
+            and am confident this bug has not been reported before
+          required: true
+  - type: textarea
+    attributes:
+      label: A clear and concise description of the bug
+      description: |
+        Tell us what happened.
+        Include a [minimal, reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) if possible.
+        Put any code blocks inside triple backticks.
+
+      value: |
+        ```py
+        your code here
+        ```
+    validations:
+      required: true
+  - type: dropdown
+    attributes:
+      label: CPython versions tested on
+      multiple: true
+      options:
+        - "3.8"
+        - "3.9"
+        - "3.10"
+        - "3.11"
+        - "3.12"
+        - "CPython main branch"
```

```suggestion
        - "CPython main branch (3.13)"
```

---

Do you think this is necessary? If we make this change, it'll be one more thing to have to update in this form on a regular basis

---

Will the form need updating anyway when main becomes 3.14 and we have to add 3.13 as its own branch?

---

> Will the form need updating anyway when main becomes 3.14 and we have to add 3.13 as its own branch/option?

Yes. (Though we could possibly automate this by adding some logic to the https://github.com/python/release-tools/blob/master/run_release.py script.)

So this would just be one extra line that we would have to update, so not a massive deal. But I'm also not sure adding `(3.13)` adds too much. _But_, don't have a particularly strong opinion either way :)

---

Alternatively this could be simplified to:
```suggestion
        - "3.13-dev"
```
Do we need to mention the branch specifically? "CPython" feels redundant.

---

`3.13-dev` isn't necessarily the same thing as the `main` branch -- we'll be doing a series of alpha releases of Python 3.13 before we get to the beta freeze, at which point we might start getting bug reports in from people who've tested with an alpha release of Python 3.13, but haven't tested with an up-to-date version of the `main` branch

---

The suggestion was for some level of conceptual symmetry -- and as Hugo said these templates will be updated annually regardless so I didn't think it hurt.

---

How's it look to you after https://github.com/python/cpython/pull/107920/commits/503004a731611eb92fb7269648d6a5755f768e61 @AA-Turner? :)

---

Hmm, so I changed the final option to `3.13 (main branch or alpha release)`. This makes that section look a little confusing when the completed form is actually rendered, though:

- https://github.com/AlexWaygood/cpython/issues/12

If we don't want "(main branch or alpha release)" in that final option, we'll need some way to distinguish between the `main` branch and alpha releases once the 3.13 alphas start being released.

---

It's unfortunate how that renders. Perhaps we should just go back to the simpler 

> "CPython main branch" 

then, as it is unambiguous? We could (later) add 3.13, when the first alpha releases are out, but that might also be confusing!

---

Yeah, "CPython main branch" might be better after all :)

(Or just "main branch")

---

Okay, for now I've switched back to "CPython main branch" :)

I think when the first 3.13 alpha is released in October, we can probably just add a "3.13 alpha release" option in between "3.12" and "CPython main branch". But we can cross that bridge when we come to it!